compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -455,6 +456,23 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleStridedSlice *nod
   return std::make_unique<kernels::StridedSlice>(input, begin, end, strides, output, params);
 }
 
+std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleSqueeze *node)
+{
+  assert(node->arity() == 1);
+
+  const Tensor *input = getInputTensor(node->input());
+  Tensor *output = getOutputTensor(node);
+
+  SqueezeParams params{};
+  assert(node->squeeze_dims().size() <= 4);
```

Comment:
optional:
I think this assert is not necessary anymore, since `SqueezeParams::squeeze_dims` accept any number of dimension now

Reply:
AFAIK TFLite implementation only supports up to 4 dimensions.